api/storage_service.cc

```diff
@@ -953,8 +953,8 @@ void set_storage_service(http_context& ctx, routes& r) {
                         ss::table_sstables tst;
                         tst.keyspace = schema->ks_name();
                         tst.table = schema->cf_name();
-
-                        for (auto sstable : *t->get_sstables_including_compacted_undeleted()) {
+                        auto sstables = t->get_sstables_including_compacted_undeleted();
+                        for (auto sstable : *sstables) {
```

Comment:
Good catch. Please mark this with a Fixes #nnnn tag (and file issue nnnn) so it can be backported.

Reply:
I've added it as issue #7605